### ISSUE TITLE
Mega-Navigation über dem Footer (Neue Portalmenü Variation)

### ISSUE THREAD

**Issue by lukasniebler** [RRZE-Webteam/FAU-Einrichtungen#1282]
Für die FAU Startseite soll es einen neuen Bereich über dem Footer geben, der ein Mega-Menü anzeigt. Das Aussehen kann dem jetzigen aufgeklappten Hauptmenü entsprechen.

## Zu klärende Fragen?
@xwolfde 

- [ ] Soll das Menü Teil des Footers sein, und auf jeder Seite erscheinen? (Macht vermutlich Sinn?)
- [ ]  Alternative Darstellung f. Portalmenü unten?

<img width="1034" alt="Bildschirm­foto 2023-05-05 um 15 50 32" src="https://user-images.githubusercontent.com/58593037/236476401-6ae80114-ae79-407f-9093-3e662a684960.png">

## Vogelperspektive:
Mockup des Menüs im Zusammenspiel mit anderen Gestaltungselementen:
![Desktop Ansicht - Beta – 15](https://user-images.githubusercontent.com/58593037/236477178-715bbbbc-db15-41d7-9731-f2cf3c8ef0a6.png)

**Comment by xwolfde** [comment]
Nein, ich würde das als neue Darstellungsoption der Portalmenu-Shortcodes machen. Der Shortcode kann dann am Ende der Seite sein, muss aber nicht.
Dann haben wir eine viel größere Flexibilität.

**Comment by lukasniebler** [comment]
Noch Anmerkung: Anders als im Hauptmenü, sollten die 4 Spalten von links nach Rechts aufgefüllt werden, wobei die linke Seite bündig mit dem linken Inhaltsrand ist.

(Zumindest was die Desktop- und Tablet-Ansicht angeht).

**Comment by xwolfde** [comment]
also flex-start statt space-between. (wenn es flex grid wäre)

**Comment by MManthey** [comment]
Mega-Navigation:

![localhost_1220-2_](https://github.com/RRZE-Webteam/FAU-Einrichtungen/assets/43784820/2d4e3da0-afda-4344-bce3-4ed7fd4dcc1a)

Option in den Page Templates ist im moment fuer alle Templates mit Portalmenues vorhanden:

![localhost_wp-admin_post php_post=2224734 action=edit classic-editor](https://github.com/RRZE-Webteam/FAU-Einrichtungen/assets/43784820/9df9a078-6e3c-4353-8bba-28dc1fd2b1b4)
